src/components/presentation/NavBar.test.tsx

```diff
@@ -101,7 +101,7 @@ describe('NavBar', () => {
       showLanguageSelector: false,
     });
     expect(screen.queryAllByRole('button')).toHaveLength(0);
-    expect(screen.queryByTestId('altinn-back-button')).toBeNull();
+    expect(screen.queryByTestId('form-back-button')).toBeNull();
```

Comment:
Personally, I think we should follow the best practices from React Testing Library. What do you think? I see that test-id was used before your changes. 🙈 

```suggestion
    expect(screen.queryByRole('button', { name: 'button-text' })).toBeNull();
```

RTL Docs has great documentation on their recommendations https://testing-library.com/docs/queries/about/#priority